PageViews/PageViews.module.less

```diff
@@ -49,10 +49,16 @@
 	}
 
 	&.dot .steps {
+		display: flex;
 		min-height: @sand-pageviews-dot-steps-area-min-height;
+		align-items: center;
+		.step {
+			line-height: @sand-pageviews-dot-steps-area-min-height;
+			height: @sand-pageviews-dot-steps-area-min-height
```

Comment:
It seems this doesn't solve the issue. As we discussed, let's adjust 'height' of past and future icons only.